app/src/main/java/com/example/mykotlincalculator21/utils/OperandUtils.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.mykotlincalculator21.utils
+
+import com.example.mykotlincalculator21.utils.NumbersUtils.ZERO_NUMBER_DOUBLE_TYPE
+import com.example.mykotlincalculator21.utils.StringUtils.EMPTY_STRING
+
+class OperandUtils {
```

Comment:
the name is not correct, Util classes are to add functionalities to other classes

Reply:
Thx for the tip, now its changed, and the class has its own package